core/templates/dev/head/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js

```diff
@@ -104,6 +104,27 @@ oppia.directive('adminDevModeActivitiesTab', [
           }
         };
 
+        $scope.reloadDummyExplorations = function(){
+          // Create a dummy exploration with random title.
+          AdminTaskManagerService.startTask();
+          $scope.setStatusMessage('Processing...');
+          for (var i = 0; i < $scope.DummyExplorationCount; ++i) {
+            var title = ['Hulk Neuroscience',
```

Comment:
Pull this outside the for loop, since it's a constant. Also, call it `var POSSIBLE_TITLES`.